Pod/Classes/Acknowledgements/AcknowledgementsViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  AcknowledgementsViewController.swift
+//  Swiftilities
+//
+//  Created by Michael Skiba on 11/21/16.
+//  Copyright © 2016 Raizlabs Inc. All rights reserved.
+//
+
+import UIKit
+
+open class AcknowledgementsViewController: UITableViewController {
```

Comment:
#pp The names of these two are too similar -- I'd use `AcknowledgementListViewController`  maybe?